Homework1Part2

```diff
@@ -0,0 +1,23 @@
+git commands: config, clone, status, add, push, commit
+
+Linux commands: mkdir, cd, pwd
+
```

Comment:
You can't copy and paset from the syllabus document. You have to write in your own words